examples/tracing/otlp/README.md

```diff
@@ -32,3 +32,24 @@ To run this example application you'll need:
 1. Select `serviceName=flipt` from the search box
 1. Click 'Run Query'
 1. You should see a list of traces to explore
+
+### Datadog UI
+
+!['Datadog Example'](../../images/datadog.png)
+
+For exporting traces from OpenTelemetry to Datadog you have to provide this piece of configuration in the `otel-collector-config.yaml`:
+
+```yaml
+exporters:
+  datadog:
+    api:
+      site: datadoghq.com
+      key: ${DD_API_KEY}
+```
+
+**Note:** The `DD_API_KEY` should be replaced with your actual api key from Datadog.
+
+You can do this under the extensions configuration. You also have to add `datadog` as an entry in `exporters` under `service.pipelines.traces.exporters`.
```

Comment:
do what under the extensions configuration?

Reply:
@markphelps Simplified the wording on this.